### PR TITLE
bump to 0.10.0

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v0.9.md [hunk 1]
```diff
@@ -30,3 +30,9 @@ N/A
 * Remove API Placement PlacementDecision ClusterSet ClusterSetBinding API version v1alpha. ([#278](https://github.com/open-cluster-management-io/registration-operator/pull/278) [@haoqing0110](https://github.com/haoqing0110))
 * Remove install mode Detached. ([#282](https://github.com/open-cluster-management-io/registration-operator/pull/282) [@zhujian7](https://github.com/zhujian7))
 * Remove clusterrole/role cleanBeforeApply code added in ocm 0.8.0. ([#283](https://github.com/open-cluster-management-io/registration-operator/pull/283) [@haoqing0110](https://github.com/haoqing0110))
+
+## v0.9.1
```
Comment (qiujian16) [inline CHANGELOG/CHANGELOG-v0.9.md:34]: shouldn't this be in release-0.9 branch

Comment (zhiweiyin318) [reply CHANGELOG/CHANGELOG-v0.9.md:34]: done

**Pull request merged**